artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -2323,6 +2324,16 @@ synchronized boolean initialisePart1(boolean scalingDown) throws Exception {
          deployAddressSettingsFromConfiguration();
       }
 
+      //fix of ARTEMIS-1823
+      if (!configuration.isPersistenceEnabled()) {
+         for (AddressSettings addressSettings : addressSettingsRepository.values()) {
+            if (addressSettings.getAddressFullMessagePolicy() == AddressFullMessagePolicy.PAGE) {
+               ActiveMQServerLogger.LOGGER.pageWillBePersisted();
```

Comment:
really? did you have any user complaining about it?

Reply:
Hi, from the the issue text, it seems, that there is currently no customer complaining about this - https://issues.jboss.org/browse/JBEAP-10498. Do you think that this warning is senseless? (on the other hand it makes sense, when persistence is turned off, paging is still persisting - which has to be..)

Reply:
I agree with you. I've changed log level to INFO.